news_items/038_organize_by_module/index.html

```diff
@@ -0,0 +1,20 @@
+<p>
+    You can now declare a list of modules for your course inside
+    of your course configuration. These are collections of
+    assessments that can belong to different sets, which get 
+    displayed together in the order that you declare them. You 
+    can think of them as chapters, sections, or topic lists in your course. 
+    If you want to group together several types of assessments 
+    (for example, two homeworks and an exam), now you can!
+</p>
+<img 
+    src="module-example.png" 
```

Comment:
In this example, could we change the names of the modules to be something that's more obviously freely chosen (not just numbered). Maybe:
* "Introductory module"
* "Another module"
* "The final module"

Reply:
@mwest1066 this example no longer exists (this was a separate course instance that I created at some point, using the Module for sorting, but we ended up deleting it). Should I just create another example?

Reply:
If it's not too hard to do then I think it would be nice. The existing example makes it look like the modules are called "Module 1", "Module 2", etc (like HWs, Exams, etc)